addon/components/au-button.hbs

```diff
@@ -12,9 +12,17 @@
     {{/unless}}
   {{/if}}
   {{#if @hideText}}
-  <span class="au-u-hidden-visually">{{yield}}</span>
+    {{#if this.loading}}
+    <span class="au-u-hidden-visually">Aan het laden</span><AuLoader @padding="small" />
+    {{else}}
+    <span class="au-u-hidden-visually">{{yield}}</span>
+    {{/if}}
   {{else}}
-  {{yield}}
+    {{#if this.loading}}
+    Aan het laden <AuLoader @padding="small" class="au-u-margin-left-tiny" />
+    {{else}}
+    {{yield}}
+    {{/if}}
   {{/if}}
   {{#if @icon}}
```

Comment:
Should the icon still be visible if the button is in the loading state? It looks a bit strange at the moment 😄.

Reply:
No, should be hidden 👍